reference/docs-conceptual/learn/shell/using-keyhandlers.md

```diff
@@ -0,0 +1,219 @@
+---
+description: This article explains how to use PSReadLine keybindings for various platforms and terminals applications.
+title: Using PSReadLine key handlers
+ms.date: 03/27/2023
+---
+# Using PSReadLine key handlers
+
+In **PSReadLine**, key handlers are configuration settings that map **PSReadLine** functions to
```

Comment:
I am not sure I like saying `In PSReadLine` as it sorta implies it a particular environment or mode of PowerShell.

Reply:
PSReadLine is the only way you get the key handlers.